### PR TITLE
Preliminary work on moving to jakarta namespace

### PR DESCRIPTION
**Pull request by smillidge**
Still needs more work when RC of JTA is available

Signed-off-by: Steve Millidge <steve.millidge@payara.fish>

### CODE REVIEW

File: api/src/main/java/jakarta/resource/cci/ConnectionFactory.java [hunk 1]
```diff
@@ -14,34 +14,32 @@
...
 
 import java.io.PrintWriter;
-import javax.resource.ResourceException;
-import javax.resource.NotSupportedException;
+import jakarta.resource.ResourceException;
+import jakarta.resource.NotSupportedException;
 
 
-/** <code>ConnectionFactory</code> provides an interface for getting
+/** *  <code>ConnectionFactory provides an interface for getting
```
[the remark this answers is not in the document]
Comment (smillidge) [reply api/src/main/java/jakarta/resource/cci/ConnectionFactory.java]: Weird I used Netbeans refactor mode. I will check the code.

**Comment by smillidge** [comment]
Corrected javadoc anomalies introduced by a NetBeans automatic refactor

File: api/src/main/java/jakarta/resource/spi/work/TransactionContext.java [hunk 2]
```diff
@@ -14,10 +14,10 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  */
 
-package javax.resource.spi.work;
+package jakarta.resource.spi.work;
 
 /**
- * A standard <code>WorkContext</code> that allows a <code>Work</code> instance
+ * A standard <code>WorkContext<code>Work instance
```
Comment (njr-11) [inline api/src/main/java/jakarta/resource/spi/work/TransactionContext.java]: End `</code>` tag was converted to begin `<code>` tag.  Also, text was lost.

**Comment by bshannon** [comment]
> Corrected javadoc anomalies introduced by a NetBeans automatic refactor

Hopefully you're filing lots of bugs against NetBeans!

**Comment by smillidge** [comment]
I hope I've caught it all now

**Pull request merged**